app/settings.coverage.json

```diff
@@ -0,0 +1,13 @@
+
+{
+  "coverage": {
+    "coverage_app_folder": "/Users/Leo/WebstormProjects/winsleague/app/",
```

Comment:
A json files is not computed, so you can't use a bash command easily. I think this way is more reliable (it's really cross platform, which is hard to achieve). Commit it one time and explain that you need to edit that path to run coverage in watch mode is fair, right ?  
I really need that path, and I can"t rely on runtime variables, so I don't know a better way